ci/tests/vpool/vpool_test.py

```diff
@@ -13,64 +13,80 @@
 # limitations under the License.
 
 from ci.tests.general import general
+from ci.tests.general.connection import Connection
 from ci.tests.general.general import test_config
 from ci.tests.backend import alba, generic
 from ovs.dal.lists.backendlist import BackendList
-from ovs.dal.lists.albanodelist import AlbaNodeList
 from ovs.extensions.generic.system import System
-from ovs.lib.albacontroller import AlbaController
-from ovs.lib.albanodecontroller import AlbaNodeController
 from ovs.lib.storagerouter import StorageRouterController
 from ovs.dal.lists.vpoollist import VPoolList
 from ci.tests.disklayout import disklayout
 
 
 VPOOL_NAME = test_config.get('vpool', 'vpool_name')
-VPOOL_NAME = 'vpool-' + VPOOL_NAME
+assert VPOOL_NAME, "Please fill out a valid vpool name in autotest.cfg file"
+
 BACKEND_NAME = test_config.get('backend', 'name')
 BACKEND_TYPE = test_config.get('backend', 'type')
+assert BACKEND_NAME, "Please fill out a valid backend name in autotest.cfg file"
+assert BACKEND_TYPE in generic.VALID_BACKEND_TYPES, "Please fill out a valid backend type in autotest.cfg file"
+
 GRID_IP = test_config.get('main', 'grid_ip')
+NR_OF_DISKS_TO_CLAIM = int(test_config.get('backend', 'nr_of_disks_to_claim'))
+TYPE_OF_DISKS_TO_CLAIM = test_config.get('backend', 'type_of_disks_to_claim')
+
+
+def add_read_write_scrub_roles(storagerouter_guid):
```

Comment:
For testing purposes i think it would be wiser to have a method which allows you to add a specific role to a partition instead of always having to add read, write and scrub
Also distributed scrubbing allows us in the framework to NOT assign a scrub role on all storagerouters